documentation/modules/exploit/multi/http/git_submodule_exec.md

```diff
@@ -0,0 +1,62 @@
+## Vulnerable Application
+
+  Git can be installed on a variety of operating systems, however
+  newer versions may contain the patch for this vulnerability.
+
+  On OSX it can be installed with the XCode command line tools:
+  ```xcode-select --install```
+  
+  On Linux it can be installed with apt:
+  ```sudo apt-get update && sudo apt-get install git```
+
+  You can check the version with ```git --version```.
+  The fix is included in the following version:
+    2.7.6, 2.8.6, 2.9.5, 2.10.4, 2.11.3, 2.12.4, 2.13.5, 2.14.1
+
+## Verification Steps
+
+  Example steps in this format:
+
+  1. Install the application
+  1. Start msfconsole
+  1. Do: ```use exploit/multi/http/git_submodule_command_exec```
+  1. Do: ```set SRVHOST [local host]```
+  1. Do: ```set LHOST [local host]```
+  1. Do: ```exploit```
+  1. Clone the malicous Git URI and its submodules
```

Comment:
malicous -> malicious

Reply:
im on heightened alert from 8888 for spelling issues

Reply:
other than that docs look good to me

Reply:
Great spot thanks :)